examples/libsodium/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.0)
+project(libsodium-example)
+
+set(HUNTER_CONFIGURATION_TYPES Release) #Release or Debug
```

Comment:
Figured out that the /p:TargetName=<library file name> cli option can be used to rename the library file name. Will use that to have Debug and Release builds at the same time.